src/main/java/ladder/domain/Players.java

```diff
@@ -0,0 +1,48 @@
+package ladder.domain;
+
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+public class Players {
+
+    private static final int MINIMUM_COUNT_OF_PLAYERS = 2;
+    private List<Player> players;
+
+    public Players(final List<String> playerNames) {
+        validateDuplicatedNames(playerNames);
+        validateNumberOfPlayers(playerNames);
+        createPlayersByName(playerNames);
```

Comment:
```suggestion
        players = createPlayersByName(playerNames);
```
이 두 코드에는 어떤 차이가 있을까요?